cstar/base/hpc_env_var_map.py

```diff
@@ -0,0 +1,100 @@
+import os
```

Comment:
This file is responsible for just the env vars that are meaningful in the the HPC envs